services/app/src/components/chessBoard/ChessBoard.jsx

```diff
@@ -8,22 +8,33 @@ const ChessBoard = () => {
   const { placeItemOnBoard, board } = useContext(ChessBoardContext);
 
   return (
-    <table id="chess-board">
-      {board.map((row, rowIndex) => (
-        <tr key={rowIndex}>
-          {row.map((tile, columnIndex) => (
-            <Tile
-              key={columnIndex}
-              tile={tile}
-              onClick={placeItemOnBoard}
-              row={rowIndex}
-              column={columnIndex}
-              boardLength={board.length}
-            />
-          ))}
+    <div className="chess-board-container">
+      <div className="rotated-outer">
+        <span className="rotated-inner axis-title">Annotation of Work</span>
+      </div>
+      <table id="chess-board">
```

Comment:
I think it has an error in inpector saying that this needs to be wrapped inside <tbody> tag.